src/OneDrive.d.ts

```diff
@@ -104,8 +104,12 @@ export declare class OneDrive extends EventEmitter {
 
   listSubscriptions(): Promise<GraphResult>;
 
+  createSubscription(resource: string, notificationUrl: string, clientState: string, changeType: string, expiresIn: number): Promise<GraphResult>;
```

Comment:
this is a bit a long argument list... maybe use an options object instead?

Reply:
Well, `resource`, `notificationUrl` and `clientState` are not "optional", so just the latter two arguments, or a `params` object instead?

Reply:
that's what I mean.. 

```js
  createSubscription({
    resource: ..., 
    notificationUrl: ..., 
    clientState: ...., 
    changeType: ...., 
    expiresIn: ...
});